modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -887,6 +903,22 @@ class InternalFFMpegRegister
     }
 };
 
+#if LIBAVUTIL_BUILD >= CALC_FFMPEG_VERSION(56,14,100)
+int CvCapture_FFMPEG::init_hw(AVCodecContext *ctx, const char *hwaccel_device)
+{
+    int err = 0;
+
+    if ((err = av_hwdevice_ctx_create(&hw_device_ctx, AV_HWDEVICE_TYPE_CUDA, hwaccel_device,
```

Comment:
What about other HW decoders: VDPAU, VAAPI, DXVA2?

Reply:
Will be implemented